### ISSUE TITLE
Probem while importing the Community Picker 

### ISSUE THREAD

**Issue by medienwerksstatt** [react-native-picker/picker#83]
# Bug report

## Environment info

`react-native info` output:

```bash
# The project name ' @react-native-community/picker' must not contain any of the following characters: [/, \, :, <, >, ", ?, *, |]. Set the 'rootProject.name'
```

**Comment by Naturalclar** [comment]
@medienwerksstatt Thanks for raising the issue!
Could you provide us with more details on where this error is happening?

**Comment by HassanRazaMuhammadTufail** [comment]
I am too facing this error

**Comment by Ailton265** [comment]
I have the same problem as well, anyone know found a solution?

**Comment by Naturalclar** [comment]
@HassanRazaMuhammadTufail @Ailton265 could you tell me which react-native version this issue is occuring?

**Comment by Ailton265** [comment]
> @HassanRazaMuhammadTufail @Ailton265 could you tell me which react-native version this issue is occuring?

@Naturalclar  on rn 0.62.2

**Comment by Coobers** [comment]
Change your settings.gradle, replace ' @react-native-community/picker' with '@react-native-community_picker' (removed space and /) and then in build.gradle  implementation project(':@react-native-community_picker')

Readme change in pull https://github.com/react-native-picker/picker/pull/158

**Comment by Naturalclar** [comment]
Fixed document in #158, thanks @Coobers!
